src/printapp/templates/base.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<html class="no-js" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    {% if title %}
+      <title>{{ title }}</title>
```

Comment:
I like having the tile `About Calvin Web Print`, but I also like `Statistics - Calvin Web Print`. I guess changing to `{{ title }} - Calvin Web Print` would be fine.